doc/user-guide.md

```diff
@@ -423,6 +423,88 @@ To implement complex deletion logic, you can add a finalizer to your Custom Reso
 deleted until you remove the finalizer (ie, after your cleanup logic has successfully run). For more information, see the
 [official Kubernetes documentation on finalizers](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers).
 
+**Example:**
+
+After the metadata name line your CR file add the reference to `finalizers`. If you are following the above example your CR file should look something like this:
+
+```sh
+$ cat deploy/crds/cache_v1alpha1_memcached_cr.yaml
+apiVersion: cache.example.com/v1alpha1
+kind: Memcached
+metadata:
+  name: example-memcached
+  finalizers:
+  - finalizer.cache.example.com
```

Comment:
The controller should add this if it needs to take actions at deletion. the user or cr should not have to set this IMO.

Reply:
I will do this change.